my-dsl.md

```diff
@@ -1,38 +1,40 @@
 # Language
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
-
+The name of the language is django.
+[Django Project](https://www.djangoproject.com/)
 
 # Domain
 _Describe the language's domain in five words._
-
+Web development
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
-
+When someone executes a program using Django, it is run as a Python program. When certain requests are made, the request is converted to a series of SQL statements that alter a database to follow a data model.
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
-
+Django is leaning towards a domain-specific language because it is specific to rapid web application development. Django is written and used for database control as it relates to web applcations.
```

Comment:
Django is used for more than database control. It's a web framework. There might be an ORM built into it for database control, but that is not its primary use.